src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -479,10 +479,10 @@ def _qa_log_filename(self):
 
     # TODO: remove patch and imports once SAS starts to populate metadata in GeoTIFF
     from unittest.mock import patch
-    import opera.util.img_utils
-    from opera.util.img_utils import MockGdal
+    import opera.util.tiff_utils
+    from opera.util.tiff_utils import MockGdal
```

Comment:
While this technically works since `tiff_utils` imports `MockGdal`, it should be referencing the new `mock_utils` module for clarity